core/jobs.py

```diff
@@ -642,7 +648,7 @@ def run(self, job_id, job_class_str, output):
 
 class GoogleCloudStorageConsistentJsonOutputWriter(
         output_writers.GoogleCloudStorageConsistentOutputWriter):
-
+    """Google Cloud Storage Consistent Output Writer."""
```

Comment:
I don't understand this, maybe explain what this class is for?

Reply:
Please take a look now.